src/screens/Dashboard/SelectFavouritesBottomSheet.tsx

```diff
@@ -142,9 +142,10 @@ const SelectFavouritesBottomSheet = ({
 
             <View>
               {updatedFavorites &&
-                updatedFavorites.map((favorite, index) => {
+                updatedFavorites.map((favorite) => {
                   return (
                     <View key={favorite.id}>
+                      <SectionSeparator />
```

Comment:
Does this mean that the first will contains a separator on the top always? Before it was like the last one doesn't include a separator at the end.

Reply:
Yes. The last one still shouldn't have a separator, but the first one should have at top. Context:
https://github.com/AtB-AS/kundevendt/issues/3105#issuecomment-1373604879

Reply:
@gorandalum Last question but is the header who should have it statically (the separator) or should the list include it a the top? Because if you do scroll now the separator will be moved along the list.

Reply:
Talked with Jorge. It is fine as it is now as long as the header-title of the section also scrolls.